src/main/java/pl/byczazagroda/trackexpensesappbackend/exception/GlobalExceptionHandlerController.java

```diff
@@ -44,10 +44,18 @@ public String handleMethodArgumentNotValidException(MethodArgumentNotValidExcept
         return e.getMessage();
     }
 
+
     @ExceptionHandler(ResourceNotFoundException.class)
     @ResponseStatus(HttpStatus.INTERNAL_SERVER_ERROR)
```

Comment:
There should be some HttpStatus from 4xx series. Maybe 404 will be proper.
Because HttpStatus.INTERNAL_SERVER_ERROR 500 mean:
> The server encountered an unexpected condition that prevented it from fulfilling the request.

https://www.webfx.com/web-development/glossary/http-status-codes/what-is-a-500-status-code/

Reply:
Done - changed to 404.